artichoke-backend/src/extn/core/time/args.rs

```diff
@@ -300,6 +334,76 @@ mod tests {
         assert_eq!(0, result.nanoseconds);
     }
 
+    #[test]
+    fn month_supports_string_values() {
+        let mut interp = interpreter();
+
+        let table = [
```

Comment:
we'll need to add some mixed case and uppercase test cases to this table.

Reply:
Added as another test